modyn/supervisor/internal/grpc_handler.py

```diff
@@ -427,11 +427,12 @@ def wait_for_training_completion(
                     # conclude/progress the current epoch
```

Comment:
could you briefly explain what the overall issue here was that caused these changes?

Reply:
Invalid answers were incorrectly handled